capstone/fitness/src/main/java/com/google/sps/servlets/ViewWorkoutServlet.java

```diff
@@ -0,0 +1,24 @@
+package com.google.sps.servlets;
+
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+* ViewWorkoutServlet will show the user the progress in the workout 
+* as well as the ability to complete sessions. It is currently not finished.
```

Comment:
Instead of "It is currently not finished.", could you leave a "TODO(GabrielGonzalezDiaz): Implement..." or something like that? This way it's easier to track what open tasks you have.